src/module_2/EDA_1.md

```diff
@@ -0,0 +1,3314 @@
+    75%          3.000000  
+    max         25.000000  
+    std          2.707693  
+             variant_id                  created_at
+    count  1.810500e+04                       18105
+    mean   3.511989e+13  2021-08-15 02:27:30.703728
+    min    3.361527e+13         2020-04-30 13:09:27
+    25%    3.382643e+13         2021-03-21 10:41:42
+    50%    3.422171e+13         2021-10-16 09:11:26
+    75%    3.448855e+13         2022-01-14 22:35:14
+    max    4.013362e+13         2022-03-14 07:49:24
+    std    2.171237e+12                         NaN
+                     id  user_id                  created_at
+    count  5.457000e+03   5457.0                        5457
+    mean   2.161881e+13    128.0  2021-12-20 11:07:10.198460
+    min    1.285856e+13    128.0         2020-05-20 13:53:24
+    25%    2.133401e+13    128.0         2021-11-13 19:52:17
+    50%    2.167062e+13    128.0         2021-12-27 13:14:57
+    75%    2.192303e+13    128.0         2022-01-30 08:35:19
+    max    2.223385e+13    128.0         2022-03-13 14:12:10
+    std    4.028679e+11      0.0                         NaN
+             variant_id        price  compare_at_price
+    count  1.733000e+03  1733.000000       1733.000000
+    mean   3.694880e+13     6.307351          7.028881
+    std    2.725674e+12     7.107218          7.660542
+    min    3.361529e+13     0.000000          0.000000
+    25%    3.427657e+13     2.490000          2.850000
+    50%    3.927260e+13     3.990000          4.490000
+    75%    3.948318e+13     7.490000          8.210000
+    max    4.016793e+13    59.990000         60.000000
+           count_people  count_adults  count_children  count_babies  count_pets
+    count    325.000000    325.000000      325.000000    325.000000  325.000000
+    mean       2.787692      2.003077        0.707692      0.076923    0.636923
+    std        1.365753      0.869577        1.026246      0.289086    0.995603
+    min        0.000000      0.000000        0.000000      0.000000    0.000000
+    25%        2.000000      2.000000        0.000000      0.000000    0.000000
+    50%        3.000000      2.000000        0.000000      0.000000    0.000000
+    75%        4.000000      2.000000        1.000000      0.000000    1.000000
+    max        8.000000      7.000000        6.000000      2.000000    6.000000
+
+
+
+```python
+#OrdersAnalysis
+
+
+import matplotlib.pyplot as plt
+#Missingvalues
+#msno.bar(orders)
+
+# Número total de órdenes
+num_orders = orders.shape[0]
+print(f"Número total de órdenes: {num_orders}")
+
+# Calcular el número de ítems por pedido
+orders['num_items'] = orders['ordered_items'].apply(len)
+
+# Promedio de ítems por pedido
+avg_items_per_order = orders['num_items'].mean()
+print(f"Promedio de ítems por pedido: {avg_items_per_order:.2f}")
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+# Histograma de la distribución de ítems por pedido
+sns.histplot(orders['ordered_items'], bins=20)
```

Comment:
This plot is impossible to read: small  no xaxis, tells nothing. What was the goal of it?